modules/renter/download.go

```diff
@@ -136,6 +132,9 @@ func newDownload(f *file, destination string) *download {
 
 		downloadFinished: make(chan error),
 	}
+	// Allocate the piece size and progress bar so that the
+	d.reportedPieceSize = d.fileSize / (d.numChunks * uint64(d.erasureCode.MinPieces()))
+	d.atomicDataReceived = d.fileSize - (d.reportedPieceSize * d.numChunks * uint64(d.erasureCode.MinPieces()))
```

Comment:
this means that downloads will show a few bytes as received before anything happens?